table.csv

```diff
@@ -1,431 +1,2246 @@
+dag-json,                                                                             ipld,          0x0129,     MerkleDAG json
+udt,                                                                                  multiaddr,     0x012d,     
+utp,                                                                                  multiaddr,     0x012e,     
+unix,                                                                                 multiaddr,     0x0190,     
+p2p,                                                                                  multiaddr,     0x01a5,     libp2p
+ipfs,                                                                                 multiaddr,     0x01a5,     libp2p (deprecated)
+https,                                                                                multiaddr,     0x01bb,     
+onion,                                                                                multiaddr,     0x01bc,     
+onion3,                                                                               multiaddr,     0x01bd,     
+garlic64,                                                                             multiaddr,     0x01be,     I2P base64 (raw public key)
+garlic32,                                                                             multiaddr,     0x01bf,     I2P base32 (hashed public key or encoded public key/checksum+optional secret)
+tls,                                                                                  multiaddr,     0x01c0,     
+quic,                                                                                 multiaddr,     0x01cc,     
+ws,                                                                                   multiaddr,     0x01dd,     
+wss,                                                                                  multiaddr,     0x01de,     
+p2p-websocket-star,                                                                   multiaddr,     0x01df,     
+http,                                                                                 multiaddr,     0x01e0,     
+json,                                                                                 serialization, 0x0200,     JSON (UTF-8-encoded)
+messagepack,                                                                          serialization, 0x0201,     MessagePack
+libp2p-peer-record,                                                                   libp2p,        0x0301,     libp2p peer record type
+x11,                                                                                  multihash,     0x1100,     
+sm3-256,                                                                              multihash,     0x00534d,     
+blake2b-8,                                                                            multihash,     0x00b201,     Blake2b consists of 64 output lengths that give different hashes

```

Comment:
The hashes are prefixed with an additional `0x00` is that intentional or a bug (the MIME types as well)?

Reply:
Sorry I don't get it. What do you mean with "how many bytes". Do you mean how many bytes it takes when encoded as varint?

Reply:
Yes.

Reply:
Shouldn't then values over 0x80 also have a 0x00 prefix?

Reply:
Hm. Yes.

Reply:
Fixed.